### PR TITLE
added timestamp to error messages

### PR DESCRIPTION
**Pull request by anttimoelsae**
Context: Currently when looking at messages on an error queue it is very difficult to decipher when they arrived

This PR:
 - adds a timestamp to metadata of message payload for messages hitting error queue
 - adding `"metadata": {"timestamp": "2022-03-16T18:38:20.972894Z"}`

Notes: could additionally show unix timestamp property of RMQ messages if there's no drawback to that

### CODE REVIEW

File: ergo/amqp_invoker.py [hunk 1]
```diff
@@ -122,19 +121,21 @@ async def run_queue_loop(self, channel_pool: aio_pika.pool.Pool, queue: aio_pika
     async def handle_message(self, message_in: Message, channel_pool: aio_pika.pool.Pool):
         try:
             async for message_out in self.do_work(message_in):
+                raise Exception('ex')
```
Comment (zachschubert) [inline ergo/amqp_invoker.py:124]:
```suggestion
```

**Review by zachschubert — CHANGES_REQUESTED** [review]
looks good otherwise

File: ergo/amqp_invoker.py (hunk 1 above)
Comment (anttimoelsae) [reply ergo/amqp_invoker.py:124]: ouch

**Review by jaechoe — APPROVED** [review]
lgtm

**Review by zachschubert — APPROVED** [review]

**Pull request merged**